client/my-sites/stats/overview.jsx

```diff
@@ -88,21 +88,22 @@ export default React.createClass( {
 			);
 
 			return overview;
-		}, this );
+		} );
 
 		return (
-			<Main wideLayout={ true }>
+			<Main wideLayout>
+				<QuerySites allSites />
```

Comment:
This PR updates the Stats overview pages, `/stats/day`.... (without selecting a web site). I don't think these stats are loaded by everyone, right? We'll be more concerned if we were using `QuerySites` in the per site stats pages.